packages/uploadthing/src/sdk/index.ts

```diff
@@ -13,14 +23,16 @@ const UT_SECRET = process.env.UPLOADTHING_SECRET;
  * await deleteFiles(["2e0fdb64-9957-4262-8e45-f372ba903ac8_image.jpg","1649353b-04ea-48a2-9db7-31de7f562c8d_image2.jpg"])
  */
 export const deleteFiles = async (fileKeys: string[] | string) => {
+  guardServerOnly();
+  const apiKey = guardApiKey();
```

Comment:
Is this used anywhere other than the fetch below?

Reply:
Nope, but I didn't wanna move the order of execution so I kept it at the beginning of the function. You wanna just in-line it straight in the fetch call?

Reply:
Yeah I was thinking to just inline it